src/components/DotNavigation/DotNavigation.st.css

```diff
@@ -0,0 +1,130 @@
+:import {
+  -st-from: '../../common/formatters.st';
+  -st-named: color, fallback, value;
+}
+
+:vars {
```

Comment:
We're using the `wix-ui-core/RadioButton`, so we should also import its style here, and use
```-st-extend: RadioButton` in the correct class.

Reply:
The thing is that wix-ui-core/RadioButton component accepts two props: checkedIcon and uncheckedIcon of React.ReactNode types. These are components with their own styles. As far as I can see, nothing mentioned in wix-ui-core/RadioButton.st.css can be used in our case.

Reply:
The thing is that <code>wix-ui-core/RadioButton</code> component accepts two props: checkedIcon and uncheckedIcon of React.ReactNode types. These are components with their own styles. As far as I can see, nothing mentioned in <code>wix-ui-core/RadioButton.st.css</code> can be used in our case.

Reply:
I have to say that I really agree with you, but as a convention we prefer adding the `extend` rule.
There might be changes in the future that we would like to use.

Reply:
Done)